src/GeniusYield/Types/Providers.hs

```diff
@@ -394,14 +402,17 @@ makeGetParameters getSlotOfCurrentBlock getProtParams getSysStart getEraHist get
 
 -- | How to query utxos?
 data GYQueryUTxO = GYQueryUTxO
-    { gyQueryUtxosAtTxOutRefs'           :: !([GYTxOutRef] -> IO GYUTxOs)
-    , gyQueryUtxosAtTxOutRefsWithDatums' :: !(Maybe ([GYTxOutRef] -> IO [(GYUTxO, Maybe GYDatum)]))
-    , gyQueryUtxoAtTxOutRef'             :: !(GYTxOutRef -> IO (Maybe GYUTxO))
-    , gyQueryUtxoRefsAtAddress'          :: !(GYAddress -> IO [GYTxOutRef])
-    , gyQueryUtxosAtAddresses'           :: !([GYAddress] -> IO GYUTxOs)
-    , gyQueryUtxosAtAddressesWithDatums' :: !(Maybe ([GYAddress] -> IO [(GYUTxO, Maybe GYDatum)]))
-    , gyQueryUtxosAtPaymentCredential'   :: !(GYPaymentCredential -> IO GYUTxOs)
-    -- ^ `gyQueryUtxosAtAddressesWithDatums'` is as `Maybe` so that if an implementation is not given, a default one is used.
+    { gyQueryUtxosAtTxOutRefs'             :: !([GYTxOutRef] -> IO GYUTxOs)
+    , gyQueryUtxosAtTxOutRefsWithDatums'   :: !(Maybe ([GYTxOutRef] -> IO [(GYUTxO, Maybe GYDatum)]))
+    -- ^ `gyQueryUtxosAtTxOutRefsWithDatu  ms'` is as `Maybe` so that if an implementation is not given, a default one is used.
```

Comment:
```suggestion
    -- ^ `gyQueryUtxosAtTxOutRefsWithDatums'` is as `Maybe` so that if an implementation is not given, a default one is used.
```

Reply:
Nice catch, don't know how this additional space got inserted.